js/libs/core.js

```diff
@@ -213,6 +301,21 @@ Fliplet.FormBuilder = (function() {
         type: Array
       };
 
+      component.props._componentName = {
```

Comment:
What is the reason for these three variables to be defined as props? Can't they be just plain variables defined at the top of core.js?